roles/register_hypervisor/tasks/main.yml

```diff
@@ -25,6 +25,7 @@
                                       --username='{{ engine_username }}'
                                       --password='{{ admin_password }}'
                                       --host='{{ inventory_hostname }}'
+                                      --host_name='{{ host_name }}'
```

Comment:
Done!
The parse_args take care if the host-name is empty string:
`
    if not args.host_name:

        args.host_name = args.host
`